### ISSUE TITLE
 Odoo V12 (MRP) : Manufacturing Order not genrated for "Make To Order"

### ISSUE THREAD

**Issue by AmitBista** [odoo/odoo#32302]
Impacted versions: 12 Enterprice

Steps to reproduce: 
1 Create Two Manufacture Product and use one of them inside BOM
2 Product Design as Manufacture and Make to Order and use it in BOM
3 Now Create MO for the Main Manufacture Product
4 So once Confirm MO, MO generated for the sub manufacture product
5 Now Update Qty from the main MO and Approve it
6 But MO is not generated for the sub manufacture product

Current behavior: MO is not generated for the sub manufacture product

Expected behavior: MO should be generate for the sub manufacture product

Video/Screenshot link (optional):
check attached video for more info
https://drive.google.com/file/d/1b0bo5PomWgnochV1Lv6lLNRYuaVA9bni/view?usp=sharing

**Comment by sswapnesh** [comment]
@AmitBista 
> 6 But MO is not generated for the sub manufacture product

I think qty of Subproduct's MO should be updated in that case. (If MO is still active)

------------------

@jbeficent  Do you have any Idea about this ? (or Any reference to OCA module)

**Comment by AmitBista** [comment]
@sswapnesh 

Yes, Subproduct's MO is active. But in the case of large number of product used in BOM. and number of sub product design with make to order. than how can you find it ? Is it right way to update Subproduct's MO one by one?

**Comment by C3POdoo** [comment]
Dear @AmitBista,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
